src/cr/cube/cube.py

```diff
@@ -312,6 +311,12 @@ def is_weighted(self):
         """True if cube response contains weighted data."""
         return self._measures.is_weighted
 
+    @lazyproperty
+    def means_subvariables(self):
+        if not self.has_means:
```

Comment:
Considering that we're trying to keep cr.cube well documented I'd like to see a docstring here :D

Reply:
done